wiring/inc/spark_wiring_ble.h

```diff
@@ -128,6 +128,14 @@ enum class BleAdvertisingEventType : uint8_t {
     SCANABLE_DIRECTED                       = BLE_ADV_SCANABLE_DIRECTED_EVT
 };
 
+enum class BlePhy : uint8_t {
+    AUTO        = 0x00,
```

Comment:
Nit: use enum names in ble_hal.h to make sure the values are consistent.

Reply:
Done

Reply:
I can see the enum values are still magic numbers, instead of using `BLE_PHYS_AUTO` for example.

Reply:
ah, understood! See new commit